docs/operator-manual/app-any-namespace.md

```diff
@@ -1,9 +1,6 @@
 # Applications in any namespace
 
-**Current feature state**: Beta
-
-!!! warning
-    Please read this documentation carefully before you enable this feature. Misconfiguration could lead to potential security issues.
+**Current feature state**:
```

Comment:
Can you remove this line too?

Reply:
I also think we should add back the warning about reading the documentation carefully. Even though the feature is stable, misuse is still quite dangerous.

Reply:
I  have updated the PR and made the suggested changes. Thanks for the input.